data/publisher/src/test/java/tech/pegasys/teku/data/publisher/MetricsDataFactoryTest.java

```diff
@@ -18,51 +18,50 @@
 import static org.mockito.Mockito.when;
 
 import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
 import java.util.ArrayList;
+import java.util.List;
 import org.hyperledger.besu.metrics.Observation;
 import org.hyperledger.besu.metrics.StandardMetricCategory;
 import org.hyperledger.besu.metrics.prometheus.PrometheusMetricsSystem;
 import org.junit.jupiter.api.Test;
 import tech.pegasys.teku.infrastructure.metrics.TekuMetricCategory;
-import tech.pegasys.teku.provider.JsonProvider;
 
 class MetricsDataFactoryTest {
 
-  private JsonProvider jsonProvider = new JsonProvider();
+  //  private final JsonProvider jsonProvider = new JsonProvider();
```

Comment:
Shouldn't leave commented out code around but I suspect we should go back to using this.